src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main>
+      <div class="card" data-qa="card">
+        <img
+          class="card__picture"
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27inches Retina 5k Monoblock"
+        >
+        </img>
+        <h1 class="card__product-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rating-text">Reviews: 5</div>
+        </div>
+        <div class="card__pricing">
+          <div class="card__pricing-text">Price:</div> <!-- if made into p or h the flex-end doesnt work -->
```

Comment:
why not just `<p>` and `align-items: center;` ?

Reply:
because I wanted to use mixing..... 🥺